### ISSUE TITLE
Calculate missing row based on subjects not in target data that are in pop_data

### ISSUE THREAD

**Issue by elimillera** [atorus-research/Tplyr#29]
```
adsl <-
  tribble(~ USUBJID, ~TRT,
          "001", "TRTA",
          "002", "TRTB")

adae <-
  tribble(~ USUBJID, ~TRT, ~ AEBODSYS,
          "001", "TRTA", "An AE")

t <- tplyr_table(adae, TRT) %>%
  set_pop_data(adsl) %>%
  add_layer(
    group_count(AEBODSYS) %>%
      set_missing_count(f_str("xx", n), missing = NA)
  )

build(t)
```
```
Should result in 
# A tibble: 2 × 4
  row_label1 var1_TRTA    ord_layer_index ord_layer_1
  <chr>      <chr>                  <int>       <dbl>
1 An AE      "1 (100.0%)"               1           1
2 missing    " 1"                       1           2
```

**Comment by mstackhouse** [comment]
So what you'll need to watch out for here is that the subject would be missing rather than NA, so essentially what you're looking for is the number of USUBJID that is in ADAE but not in ADSL for whatever grouping you're examining. 

```r
adsl <- haven::read_xpt(url("https://github.com/phuse-org/TestDataFactory/raw/main/Updated/TDF_ADaM/adsl.xpt"))
adae <- haven::read_xpt(url("https://github.com/phuse-org/TestDataFactory/raw/main/Updated/TDF_ADaM/adae.xpt"))

sum(!(adsl$USUBJID %in% unique(adae$USUBJID)))
```

**Comment by mstackhouse** [comment]
Closed via #174
